internal/service/timestreaminfluxdb/db_instance.go

```diff
@@ -196,8 +110,7 @@ func (r *resourceDbInstance) Schema(ctx context.Context, req resource.SchemaRequ
 				},
 				Description: `The name of the initial InfluxDB bucket. All InfluxDB data is stored in a bucket. 
 					A bucket combines the concept of a database and a retention period (the duration of time 
-					that each data point persists). A bucket belongs to an organization. This argument is optional. 
```

Comment:
Is there a reason to remove the default value description?

Reply:
I want all descriptions to match the AWS SDK Go. Terraform should communicate default values on its own, though I will double check this.

Reply:
Terraform does not communicate defaults or optional arguments when deploying or planning deployments. The comment above `StringAttribute.Description` reads:
```
	// Description is used in various tooling, like the language server, to
	// give practitioners more information about what this attribute is,
	// what it's for, and how it should be used. It should be written as
	// plain text, with no special formatting.
```

In this case, `StringAttribute.Optional`, `StringAttribute.Required`, and `StringAttribute.Validators` provide the information that I had previously included in the description.